dialog-el.html

```diff
@@ -204,15 +204,34 @@
       if (focusableChildren[0]) focusableChildren[0].focus();
     }
 
-    /**
-     * Executes querySelector on the current document
-     */
     var querySelector = function(selector) {
       var script = document._currentScript || document.currentScript;
       return script.ownerDocument.querySelector(selector);
     };
 
     /**
+     * Gets top and left coordinates relative to document.
+     * http://stackoverflow.com/questions/5598743/finding-elements-position-relative-to-the-document
+     */
+    function getCoords(elem) {
+      var box = elem.getBoundingClientRect();
+
+      var body = document.body;
+      var docEl = document.documentElement;
+
+      var scrollTop = window.pageYOffset || docEl.scrollTop || body.scrollTop;
+      var scrollLeft = window.pageXOffset || docEl.scrollLeft || body.scrollLeft;
+
+      var clientTop = docEl.clientTop || body.clientTop || 0;
+      var clientLeft = docEl.clientLeft || body.clientLeft || 0;
+
+      var top  = box.top +  scrollTop - clientTop;
+      var left = box.left + scrollLeft - clientLeft;
+
+      return { top: Math.round(top), left: Math.round(left) };
```

Comment:
The calculation of `top` and `left` here don't take into account the dialog's `offsetParent` position.